test/libsolidity/semanticTests/multiSource/external/external.sol

```diff
@@ -0,0 +1,11 @@
+// Exclude From Tests
```

Comment:
Could we potentially use something in the path for this, like a dot prefix: `semanticTests/multiSource/.external/external.sol`?

Reply:
I did not try this yet, but I would say it should just work. However, I'm not sure, will check that tomorrow.

Reply:
So lets do underscore (`_`) as the prefix.